--- ./viewer/svmnode.cpp.fix	2008-08-06 13:51:11.000000000 +0200
+++ ./viewer/svmnode.cpp	2008-08-06 13:51:52.000000000 +0200
@@ -27,6 +27,7 @@
 #include "svmnode.h"
 
 #include <iostream>
+#include <cstring>
 
 #include "scrollview.h"
 
--- ./viewer/scrollview.cpp.fix	2008-08-06 13:50:11.000000000 +0200
+++ ./viewer/scrollview.cpp	2008-08-06 13:50:44.000000000 +0200
@@ -34,7 +34,8 @@
 #include <utility>
 #include <algorithm>
 #include <vector>
-#include <string>
+#include <cstring>
+#include <climits>
 
 #include "svutil.h"
 
--- ./viewer/svutil.cpp.fix	2008-08-06 13:52:26.000000000 +0200
+++ ./viewer/svutil.cpp	2008-08-06 13:53:32.000000000 +0200
@@ -37,7 +37,8 @@
 #endif
 
 #include <iostream>
-#include <string>
+#include <cstring>
+#include <stdlib.h>
 
 const int kBufferSize = 65536;
 const int kMaxMsgSize = 4096;
